docker-images/kafka/scripts/kafka_run.sh

```diff
@@ -19,16 +19,13 @@ export GC_LOG_ENABLED="false"
 # ... but enable equivalent GC logging to stdout
 export KAFKA_GC_LOG_OPTS="-verbose:gc -XX:+PrintGCDetails -XX:+PrintGCDateStamps -XX:+PrintGCTimeStamps"
 
-if [ -z "$KAFKA_LOG_LEVEL" ]; then
-  KAFKA_LOG_LEVEL="INFO"
-fi
 if [ -z "$KAFKA_LOG4J_OPTS" ]; then
-  export KAFKA_LOG4J_OPTS="-Dlog4j.configuration=file:$KAFKA_HOME/config/log4j.properties -Dkafka.root.logger.level=$KAFKA_LOG_LEVEL,CONSOLE"
+  export KAFKA_LOG4J_OPTS="-Dlog4j.configuration=file:$KAFKA_HOME/config/log4j.properties"
```

Comment:
The old `log4j.properties` files are still in `docker-images` although they are not used anymore.

Reply:
Deleted.